static/js/textbox.html

```diff
@@ -12,6 +12,36 @@
             return vars;
         }
 
+        function disableNavButtons(){
+            var next_buttons = window.parent.document.getElementsByClassName('button-next');
+            var previous_buttons = window.parent.document.getElementsByClassName('button-previous');
+            var i;
+            for (i = 0; i < next_buttons.length; i ++){
```

Comment:
Here we manipulate with nextButtons array and previousButtons array so forEach method isn't suitable.